### DISCUSSION TITLE
Shelly EM

### DISCUSSION THREAD

**Discussion by rjmontenegro** [arendst/Tasmota#19773] [Q&A]
Hello everybody. 
For no reason to justify it, one of Shelly EM's circuits stopped reading the values. I changed the clamps to test and the problem appears to be with the shelly and not the clamps. The clamp on the circuit that was not working, works without problems on the other circuit. The opposite does not occur. Has anyone gone through a similar situation? Can you help? 
Thanks

**Comment by barbudor** [comment]
Have you compared the configuration and template of both?
An unusual situation could have resets some parameters such as removing the gpios configuration

**Reply by rjmontenegro** [reply]
The strangest thing is that this happens in the same Shelly EM. Shelly EM have two channels/circuits and one of them simply stopped working.
